### ISSUE TITLE
https://app.antshares.org/ is not working: not sure how to move to new web wallet

### ISSUE THREAD

**Issue by futurehelen** [neo-project/neo#60]
peter lin had a guide but the select wallet file field 
<img width="1440" alt="ss" src="https://user-images.githubusercontent.com/8387924/29681943-34401208-88be-11e7-9ea6-f89ad8d34c71.png">
functionality is not working

**Comment by PeterLinX** [comment]
Hi @futurehelen, the wallet's  default name is "wallet", you need to input your password to open it. Everything looks fine in picture you sent, please tell me more details about your question.

**Comment by futurehelen** [comment]
hi Peter
no worries. I moved it to the neon wallet. However, The neon desktop wallet
(mac) is having minor issues now. it shows me my balance, the gas is
working, but I've tried twice to send my antshares and gas and both times
it hasn't worked: not a big deal, it's not like the shares/gas got lost in
transit. they're all sitting in my account but it can't send.

note: I do know that it did work at one point because I once sent 5 gas
from the neon wallet.

cheers,
helen

**Comment by DrDanP** [comment]
Hi futurehelen...where did you find the other neon wallet? Did you have to set it up yourself? I still have not been able to transfer my NEO from my app.antshares.org wallet.
Dan

**Comment by futurehelen** [comment]
you find it on the neo site: its the city of zion Neon wallet.  you don't
have to transfer anything.  just download it, then type in your private key
and it opens up the same account and show you your shares and gas and you
can claim gas. this one, you don't have to type in your old password.

so, in essence, no transfer.

I also managed to successfully send the gas I had earned out of this new
wallet.
But the last two times i've tried to send either gas or shares, it hasn't
worked. the shares and gas just stay in the wallet. So I guess...I'm
hodling whether I like it or not. haha...
